task.sql

```diff
@@ -2,10 +2,15 @@
 USE ShopDB; 
 
 -- Some data should be created outside the transaction (here)
+INSERT INTO Orders (ID, CustomerID, Date)
+	VALUES (1, 1, '2024-01-31');
```

Comment:
No need to set up the value of the ID. It's auto-increment and set up automatically